test/e2e/app/app.go

```diff
@@ -129,7 +135,10 @@ func DefaultConfig(dir string) *Config {
 // 2 - blob is empty ([]byte{})
 // 3 - blob is variable length string that contains "BLOBXXX" where XXX is height multiplied by 0x80 in hex
 // 4 - blob is the size of MaxBlobSizeBytes and contains height truncated into two bytes repeated.
-func blobOracle(height int64) ([]byte, bool) {
+func blobOracle(height int64, blobMaxBytesUpdateHeight int64) ([]byte, bool) {
+	if height <= blobMaxBytesUpdateHeight+2 || blobMaxBytesUpdateHeight == -1 {
```

Comment:
Why "+2"? (just trying to understand)

Reply:
Also, if "-1" means the same as for vote extensions and PBTS, then I don't fully get that condition either